src/rdpc/variantCallingAnalysesProcessor.ts

```diff
@@ -173,50 +186,58 @@ const mergeAllDonors = (
         ...info.sangerVC,
       ];
       merged[donorId].mutect = [...merged[donorId].mutect, ...info.mutect];
+      merged[donorId].openAccess = [
+        ...merged[donorId].openAccess,
+        ...info.openAccess,
+      ];
     } else {
       merged[donorId] = info;
     }
   });
 };
 
-const convertAnalysis = (
-  analyses: Analysis[]
-): StringMap<SangerAndMutectInfo> => {
-  const result: StringMap<SangerAndMutectInfo> = {};
+const convertAnalysis = (analyses: Analysis[]): StringMap<WorkflowInfo> => {
+  const result: StringMap<WorkflowInfo> = {};
   analyses.forEach((analysis) => {
-    const infoMapPerAnalysis = analysis.donors.reduce<
-      StringMap<SangerAndMutectInfo>
-    >((infoAccumulator, donor) => {
-      let workflowName = "";
-      if (analysis.workflow && analysis.workflow.workflowName) {
-        workflowName = analysis.workflow.workflowName.toLocaleLowerCase();
-      } else {
-        logger.warn(`Incomplete RDPC Data: analysis id: ${analysis.analysisId} does not have 'workflow' or 'workflowname',
+    const infoMapPerAnalysis = analysis.donors.reduce<StringMap<WorkflowInfo>>(
+      (infoAccumulator, donor) => {
+        let workflowName = "";
+        if (analysis.workflow && analysis.workflow.workflowName) {
+          workflowName = analysis.workflow.workflowName.toLocaleLowerCase();
+        } else {
+          logger.warn(`Incomplete RDPC Data: analysis id: ${analysis.analysisId} does not have 'workflow' or 'workflowname',
         this analysis won't be included in the donor's aggregated stats.`);
-      }
-
-      const info: SangerAndMutectInfo = {
-        sangerVC: [],
-        mutect: [],
-      };
-
-      const workflowData = {
-        analysisId: analysis.analysisId,
-        workflowName: workflowName,
-        firstPublishedAt: analysis.firstPublishedAt,
-      };
-
-      if (workflowName.includes("mutect2")) {
-        info.mutect.push(workflowData);
-      }
-
-      if (workflowName.includes("sanger")) {
-        info.sangerVC.push(workflowData);
-      }
-
-      infoAccumulator[donor.donorId] = info;
-      return infoAccumulator;
-    }, {});
+        }
+
+        const info: WorkflowInfo = {
+          sangerVC: [],
+          mutect: [],
+          openAccess: [],
+        };
+
+        const workflowData = {
+          analysisId: analysis.analysisId,
+          workflowName: workflowName,
+          firstPublishedAt: analysis.firstPublishedAt,
+        };
+
+        if (workflowName.includes(WORKFLOW_NAMES.MUTECT)) {
+          info.mutect.push(workflowData);
+        }
+
+        if (workflowName.includes(WORKFLOW_NAMES.SANGER)) {
+          info.sangerVC.push(workflowData);
+        }
+
+        if (workflowName.includes(WORKFLOW_NAMES.OPEN_ACCESS)) {
```

Comment:
the `workflowName` here will not include `open access` because in order to get open access workflow first publish date, the analysisType used in the fetch query should be `variant_processing` , but in `fetchVariantCallingAnalyses.ts` line 59, it's using `variant_calling` as analysis type, which will give you sanger and mutect workflow instead of open access workflow. So you might want to refactor a bit to make sure `variant_processing` is used in the query filter.

Reply:
Thanks for catching this!